src/utils/getTwitterCardTags.js

```diff
@@ -23,10 +23,22 @@ const schemas = {
   player: TWITTER_PLAYER_CARD_SCHEMA,
   app: TWITTER_APP_CARD_SCHEMA,
   summary: TWITTER_SUMMARY_CARD_SCHEMA,
-  summary_with_large_image: TWITTER_SUMMARY_CARD_SCHEMA,
+  summary_large_image: TWITTER_SUMMARY_CARD_SCHEMA,
 };
 
 const getTwitterCardTags = (tags) => {
+  if (tags.card === 'summary_with_large_image') {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'summary_with_large_image is deprecated and will be removed in the next major release. Please use summary_large_image instead. See https://github.com/americanexpress/react-seo/issues/24 for more information'
+    );
```

Comment:
Could also use `npm i invariant` to warn. It will be silenced in production using that utility. Its written and used by the React team. 

```
invariant(
      'summary_with_large_image is deprecated and will be removed in the next major release. Please use summary_large_image instead. See https://github.com/americanexpress/react-seo/issues/24 for more information',
    tags.card !== 'summary_with_large_image');
```

Reply:
ah cool, haven't used that before. Thank you sir.

Reply:
Looks like `invariant` with throw rather than warn.  I wanted to avoid breaking existing usage of `summary_with_large_image`